services/tenant-ui/frontend/src/components/reservation/Reserve.vue

```diff
@@ -219,5 +173,42 @@ const handleSubmit = async (isFormValid: boolean) => {
     console.error(err);
     toast.error(`Failure making request: ${err}`);
   }
+
+  console.log('form is valid');
 };
 </script>
+<style scoped lang="scss">
```

Comment:
Can we add a bit of top margin here between inputs so the form is spaced out as others are?

Reply:
Sure thing. You mean like the previous reservation UI?

Reply:
Right, just some space like other forms have (not sure we're entirely consistent with exact spacing accross the app) but they're generally spaced out more.
Ex:

![image](https://github.com/bcgov/traction/assets/17445138/1a85699b-d103-4b9d-9dbf-8e4fc872da73)

Reply:
![Screenshot from 2023-09-08 12-03-31](https://github.com/bcgov/traction/assets/479074/723ee31a-e406-46e5-88d1-7b753bcd46d6)